src/components/Notification/StatusNotification.jsx

```diff
@@ -19,14 +21,22 @@ import StatusMessage from './StatusMessage';
  */
 
 const StatusNotification = ({ notification, statusType, defaultMessage, locationUrl = '' }) => (
-  <dl className="display">
-    <dt>{statusType}:</dt>
+  <Typography
+    sx={{
+      marginTop: 3,
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'center'
+    }}
+  >
+    <>{statusType}:</>
```

Comment:
Considering it was suppose to be a `<dt>` element, you could wrap this with `<Typography>`